homeassistant/components/maxcube/config_flow.py

```diff
@@ -0,0 +1,35 @@
+"""Adds config flow for maxcube."""
```

Comment:
100% test coverage is required for `config_flow`. Please see other integrations to see how it is done.

Reply:
done